big-practice/src/js/dashboard.js

```diff
@@ -30,15 +32,15 @@ const showErrorMessage = (input, msg) => {
 /**
  * Hide modal
  */
-const hideModal = () => {
-  modal.classList.remove('modal-show');
+const hideModal = (hideModel) => {
```

Comment:
model?
Btw, do we need hide? Because function name already have hide in its name